web/src/lib/components/alert/alert.tsx

```diff
@@ -0,0 +1,49 @@
+import { h, Component } from 'preact';
+import Icon from '../icon';
+
+interface Props {
+  text: string;
+  active: boolean;
+}
+
+interface State {
+  active: boolean;
+}
+
+export default class Alert extends Component<Props, State> {
+  constructor(props: Props) {
+    super(props);
+
+    this.onClick = this.onClick.bind(this);
+  }
+
+  state: State = {
+    active: false,
```

Comment:
this should be initialized with the given `active` value passed to the constructor :)